### DISCUSSION TITLE
Pet Licenses "spawn" Eater of Souls and Angry Bones

### DISCUSSION THREAD

**Discussion by grechnevi** [Pryaxis/TShock#1805] [Q&A] [answered]
It says in the chat
- for the Cat License "Username summoned Eater of Souls!"
- for the Dog License "Username summoned Angry Bones!"

But pets come after some amount of time.

**Comment by QuiCM** [comment]
That is mildly hilarious. We'll have a look

**Comment by Olink** [comment, marked as answer]
I've created an issue to track this.
https://github.com/Pryaxis/TShock/issues/1904
